kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTask.java

```diff
@@ -234,10 +234,14 @@ private RowToInsert getRecordRow(SinkRecord record) {
     }
   }
 
-  private Map<TableId, List<SinkRecord>> partitionRecordsByTable(
-      Collection<SinkRecord> records) {
+  private Map<TableId, List<SinkRecord>> partitionRecordsByTable(Collection<SinkRecord> records) {
     Map<TableId, List<SinkRecord>> tableRecords = new HashMap<>();
     for (SinkRecord record : records) {
+      if (recordEmpty(record)) {
+        // ignore it
+        logger.debug("ignoring empty record value for key: " + record.key().toString());
```

Comment:
Keys can be null in Kafka as well. It would be super weird to send a null key and null value, but it's possible.